modules/containerregistry.bicep

```diff
@@ -1,25 +1,33 @@
 // This template is used to create a Container Registry with private endpoint.
```

Comment:
I liked the comments you put in the other modules - it looks much cleaner - you could add that here as well.

`// Creates a Container Registry with Private Link Endpoint`